api/types/image/image_inspect.go

```diff
@@ -111,7 +111,11 @@ type InspectResponse struct {
 
 	// GraphDriver holds information about the storage driver used to store the
 	// container's and image's filesystem.
-	GraphDriver storage.DriverData
+	GraphDriver storage.DriverData `json:"GraphDriver,omitempty"`
+
+	// StorageDriver holds information about the storage driver used to store the
+	// container's and image's filesystem.
+	StorageDriver storage.Driver `json:"StorageDriver,omitempty"`
```

Comment:
I don't think this is future proof, containerd can use multiple storage drivers, one may just be the default

Reply:
This one is for image inspect; you mean an image that is present in multiple snapshotters?

Reply:
For images, it could be unpacked into zero or more snapshotter. If we want to start favoring the content store as the source of truth for image storage and treat the snapshotter as a cached of the unpacked value, then it may be confusing to generically say this is how the image is "stored".

Reply:
Yeah, so we either should omit it entirely, or go all the way and enumerate all snapshots.

But that would  bring back the discussion if such unpacked content belongs to the "image", so probably content store only would make more sense to track.

Some of that might impact how we present sizes in "docker image ls --tree" and "docker system df".

I guess for containers, it's less ambiguous, as they would always be tied to a specific snapshotter. We must still verify our code if we always use the container's property to reference the snapshotter though, as I recall early implementations tended to use the daemon's default as source of truth (thus not accounting for snapshotters to be differing per container)